### PR TITLE
Add Support to FCoE Network

### PR DESCRIPTION
**Pull request by yuvirani**
### Check List
- [x] New functionality includes sanity testing.
  - [x] All sanity tests pass. (`$ ansible-test sanity`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).

### CODE REVIEW

File: plugins/modules/oneview_fcoe_network_facts.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2016-2019) Hewlett Packard Enterprise Development LP
```
Comment (VenkateshRavula) [inline plugins/modules/oneview_fcoe_network_facts.py:4]: change copyright

Comment (yuvirani) [reply plugins/modules/oneview_fcoe_network_facts.py]: Done

File: roles/oneview_fcoe_network_facts/meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+galaxy_info:
+  role_name: oneview_fcoe_network
```
Comment (VenkateshRavula) [inline roles/oneview_fcoe_network_facts/meta/main.yml:3]: role_name should be oneview_fcoe_network_facts

File: roles/oneview_fcoe_network_facts/meta/main.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  author: Yuvarani Chidambaram
+  description: Ansible role to manage OneView FCoE Network resources.
```
Comment (VenkateshRavula) [inline roles/oneview_fcoe_network_facts/meta/main.yml:5]: mentions facts in description

File: tests/unit/test_oneview_fcoe_network_facts.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2016-2019) Hewlett Packard Enterprise Development LP
```
Comment (VenkateshRavula) [inline tests/unit/test_oneview_fcoe_network_facts.py:4]: change copyright year

File: roles/oneview_fcoe_network_facts/defaults/main.yml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+---
+# defaults file for oneview_fcoe_network_facts
+network_name: 'Ansible Test FCoE Network'
+config: "~/.ansible/collections/ansible_collections/hpe/oneview/roles/oneview_fcoe_network/files/oneview_config.json"
```
Comment (VenkateshRavula) [inline roles/oneview_fcoe_network_facts/defaults/main.yml:4]: also config path should have oneview_fcoe_network_facts

**Pull request merged**